### PR TITLE
[CF-225] Correct using of gateway variable

### PR DESCRIPTION
**Pull request by SVilgelm**

### CODE REVIEW

**Comment by MirantisWorkloadMobility** [comment]
Consider resolving pylint issues: \
``` \
************* Module cloudferrylib.os.actions.check_ssh
cloudferrylib/os/actions/check_ssh.py:27: [W0613(unused-argument), CheckSSH.run] Unused argument 'info'

**Comment by MirantisWorkloadMobility** [comment]
PEP8 check success, Unittests success

File: cloudferrylib/os/actions/cinder_database_manipulation.py [hunk 1]
```diff
@@ -380,11 +379,9 @@ def fix_metadata(self, data):
 
     def _run_cmd(self, cloud, cmd):
         runner = _remote_runner(cloud)
-        with settings(gateway=cloud[CFG].get(SSH_HOST),
-                      connection_attempts=self.ssh_attempts):
-            output = runner.run(cmd)
-            res = output.split('\r\n')
-            return res if len(res) > 1 else res[0]
+        output = runner.run(cmd)
+        res = output.split('\r\n')
```
Comment (roman-verchikov) [inline cloudferrylib/os/actions/cinder_database_manipulation.py]: `splitlines()`?

**Comment by plomakin** [comment]
+1, jobs fails because of https://github.com/MirantisWorkloadMobility/CloudFerry/pull/839

**Comment by vladiskuz** [comment]
+1, Tested on my environment. Migration successfully finished with this patch.

**Comment by plomakin** [comment]
@roman-verchikov please, review

**Comment by roman-verchikov** [comment]
+1, can be merged

**Pull request merged**